### PR TITLE
Feat: Added not found page, document custom hook, scroll to top hook

### PR DESCRIPTION
**Pull request by sainath7878**
Added not found page, document custom hook, scroll to top hook
Files to review: 

- [src/hooks/useDocumentTitle.jsx](https://github.com/sainath7878/joddquiz/pull/5/files#diff-3539bcc9ca326f0edbdaca777e8e1cc054798b884d489927e828e58e8f054b24)
- [src/hooks/useScrollToTop.jsx](https://github.com/sainath7878/joddquiz/pull/5/files#diff-2f30b311d907f9d1a1d58fbbd37a42dea9ca2b29fd4333cd6f32f212d25e399e)
- [src/pages/notFound/NotFound.jsx](https://github.com/sainath7878/joddquiz/pull/5/files#diff-de85933229bff783692e90bd7453bf4c803955e6858946ec8f9316e352a1da0a)

### CODE REVIEW

**Review by BalajiHarinath — APPROVED** [review]
Everything looks good. You can merge.

File: src/App.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ function App() {
             <Route path="/signin" element={<SignIn />} />
             <Route path="/signup" element={<SignUp />} />
           </Route>
+          <Route path="*" element={<NotFound />} />
```
Comment (anik31) [inline src/App.js:43]: appreciation - good that you've added 404 page

File: src/hooks/useDocumentTitle.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { useEffect } from "react";
+
+const useDocumentTitle = (title) => {
+  useEffect(() => {
+    document.title = title;
+  }, []);
+};
```
Comment (anik31) [inline src/hooks/useDocumentTitle.jsx:7]: appreciation - liked that you've implemented this hook

File: src/hooks/useScrollToTop.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+
+const useScrollToTop = () => {
+  useEffect(() => {
+    window.scrollTo({
+      top: 0,
+      behavior: "smooth",
+    });
+  }, []);
```
Comment (anik31) [inline src/hooks/useScrollToTop.jsx:9]: suggestion - here you can give a dependency of pathname such that it does not run once the component gets rendered. you can reproduce this issue by going to questions page, scroll to bottom and then click on next question

**Review by anik31 — APPROVED** [review]
Nice work, everything looks good. Code is very clean & good implementation of hooks.

**Pull request merged**